operator/internal/controller/check-psql.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+count=0
+
+until [ $count -ge 3 ]; do
+  psql -U postgres -c 'SELECT 1' && count=$((count + 1))
+  echo "Waiting for postgres to be ready..."
+  sleep 5
```

Comment:
```suggestion
  sleep 1
```

Reply:
I think this will save some startup time

Reply:
Yeah its tricky... the Postgres container temporarily starts its Postgres server to do some bootstrapping, shuts down, and then starts back up again.

So I've found the time needs to be longer in order to not accidentally consider Postgres "ready" too soon.

I wonder if instead I can look for the presence of a folder/file to indicate bootstrapping is done, before doing a quick health check.

Reply:
Interesting. Maybe it's worth querying `pg_isready`? I believe that should respond only once the server is ready

Reply:
Great suggestion - testing that now